shimmingtoolbox/read_nii.py

```diff
@@ -0,0 +1,73 @@
+import json
+import math
+import nibabel as nib
+import numpy as np
+import os
+
+
+def read_nii(nii_path):
```

Comment:
is this function needed? `nibabel.load()` outputs the image + header (which contains the metadata)

Reply:
I am going to remove it. It is just convenient since it was already programmed with the verifications.